_posts/2022-06-06-homebrew-3.5.0.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 3.5.0
+author: MikeMcQuaid
+redirect_from: /blog/3.5.0/
+---
+Today I'd like to announce Homebrew 3.5.0. The most significant changes since 3.4.0 are improved `brew update` behaviour and Homebrew (on macOS) requiring at least OS X El Capitan (10.10).
```

Comment:
```suggestion
Today I'd like to announce Homebrew 3.5.0. The most significant changes since 3.4.0 are improved `brew update` behaviour and Homebrew (on macOS) requiring at least OS X El Capitan (10.11).
```